lib/storage/storage_test.go

```diff
@@ -1840,7 +1859,7 @@ func TestStorageRotateIndexDB_Search(t *testing.T) {
 // testRotateIndexDB checks that storage handles gracefully indexDB rotation
 // that happens concurrently with some operation (ingestion or search). The
 // operation is expected to finish successfully and there must be no panics.
-func testRotateIndexDB(t *testing.T, mrs []MetricRow, op func(s *Storage)) {
+func testRotateIndexDB(t *testing.T, _, _ uint32, mrs []MetricRow, op func(s *Storage)) {
```

Comment:
to minimize diff with `issue-7599-cluster` I passed `accountID/projectID`, but ignore them for now.